analyses/01_sample_distribution_analysis/01-tumor-descriptor-and-assay-count-plot.Rmd

```diff
@@ -0,0 +1,504 @@
+        theme_classic()+
+        ylab("Count cancer_group") +
+        ggtitle("cancer_group vs experimental_strategy") +
+        guides(color=guide_legend(override.aes=list(size=3))) +
+        theme(axis.text.x = element_text(angle = 90, vjust = 0.5, hjust = 1))
+        print(p)
+#ggsave("Barplot_cancer_group_experimental_strategy_PBTA_matched_DNA-RNA.pdf", plot=p, path=plots_dir, device="pdf", height=7, width=10)
+
+```
+
+## By cancer_group - all disease stages
+
+
+```{r plot3}
+count$tumor_descriptor <- factor(x = count$tumor_descriptor, levels = c("Diagnosis", "Progressive", "Recurrence", "Deceased", "Second Malignancy", "Metastatic", "Not Reported", "Residual", "Unavailable"))
+
+
+p <- ggplot(count, aes(x=cancer_group, fill=tumor_descriptor))+
+        geom_bar()+
+        scale_fill_manual(values=col9) +
+        theme_classic()+
+        ylab("Count cancer_group") +
+        ggtitle("cancer_group vs tumor_descriptor") +
+        guides(color=guide_legend(override.aes=list(size=3))) +
+        theme(axis.text.x = element_text(angle = 90, vjust = 0.5, hjust = 1))
+        print(p)
+#ggsave("Barplot_cancer_group_tumor_descriptor_PBTA_matched_DNA-RNA.pdf", plot=p, path=plots_dir, device="pdf", height=7, width=10)
+
+```
+
+# Summary
+
+In the following section, we will provide summary table across different disease stages, histologies, and patients samples.
+
+## Number of each assay
+
+First, we’ll examine how many of each type of assay we have (tumors only). This information is stored in the experimental_strategy column.
+
+```{r}
+tumor_df %>%
+  group_by(experimental_strategy) %>%
+  tally() %>%
+  arrange(desc(n)) %>% 
+  regulartable() %>%
+  fontsize(size = 12, part = "all")
+
+```
+
+
+
+# Diagnosis vs. any other point in time (recurrence, progressive etc)
+
+We’ll look at the breakdown of the tumor_descriptor column, separating the genomic assays from transcriptomic assays.
+
+```{r}
+tumor_descriptor_df <- tumor_df %>%
+  select(Kids_First_Participant_ID,
+         experimental_strategy,
+         tumor_descriptor,
+         cancer_group) %>%
+  arrange(Kids_First_Participant_ID)
+```
+
+
+## Genomic assays
+Setting aside the Panel sample for the moment and only looking at WXS and WGS assays. We’re collapsing the different values in tumor_descriptor to form a single descriptor when there are multiple types of tumors from the same individual.
+
+
+```{r}
+genomic_df <- tumor_descriptor_df %>%
+  filter(experimental_strategy %in% c("WGS", "WXS")) %>%
+  group_by(Kids_First_Participant_ID) %>%
+  summarize(descriptors = paste(sort(unique(tumor_descriptor)),  
+                                collapse = ", "),
+            experimental_strategy = paste(sort(unique(experimental_strategy)),
+                                          collapse = ", "),
+            cancer_group = paste(sort(unique(cancer_group)),
+                                          collapse = ", "))
```

Comment:
want to get rid of unique here to make sure you see _all_ of the samples available.

Reply:
group_by(Kids_First_Participant_ID, match_id, cancer_group) %>%